addons/main/stringtable.xml

```diff
@@ -401,6 +401,22 @@
             <English>Allow self revive</English>
             <Polish>Zezwól na samodzielne podniesienie</Polish>
         </Key>
+        <Key ID="STR_diw_armor_plates_main_visibleBleedoutTimer">
+            <English>Show bleedout timer to</English>
+            <Polish>Komu powinno się pokazać odliczanie do śmierci?</Polish>
```

Comment:
In general if those are machine translated do not add them. Imo better to have english only and not something that is garbled sentences in another language.